app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(name=person["name"], age=person["age"])
+                   for person in people]
+    for person in people:
+        main_person = Person.people[person["name"]]
+        if person.get("husband"):
+            husband_person = Person.people[person["husband"]]
+            main_person.husband = husband_person
+        elif person.get("wife"):
+            wife_person = Person.people[person["wife"]]
+            main_person.wife = wife_person
```

Comment:
This code can lead to a `KeyError` if the person's husband or wife is not in the `Person.people` dictionary. To avoid this, you should check if the husband or wife is in the dictionary before attempting to access it.

Reply:
для того після цього фрагмену і стоїть виклик списку person_list